conda-recipe/meta.yaml

```diff
@@ -23,7 +23,10 @@ requirements:
   - python >=3.9
   - numpy
   - scipy
+  - xraylib
   - xraydb
+  - periodictable
+  - ophyd
```

Comment:
nitpick: the module dependencies are in an arbitrary order rather than alphabetized